src/main/java/org/ladysnake/effective/EffectiveConfig.java

```diff
@@ -93,10 +99,10 @@ public class EffectiveConfig extends MidnightConfig {
 	public static boolean improvedDragonFireballsAndBreath = true;
 
 	@Entry(category = improvedEffects)
-	public static boolean improvedSpectralArrows = true;
+	public static boolean improvedGlowSquidParticles = true;
 
 	@Entry(category = improvedEffects)
-	public static boolean improvedGlowSquidParticles = true;
+	public static AllayTrailOptions spectralArrowTrails = AllayTrailOptions.BOTH;
```

Comment:
Should probably rename the enum to `TrailOptions` if it's going to be reused

Reply:
yeah probably, i didn't wanna do anything too drastic but it'd make more sense